### ISSUE TITLE
Side-by-side comparison with pandas

### ISSUE THREAD

**Issue by jbe456** [atoti/atoti#46]
## Description

Suggested by a first time user:
> I imagine people using this would come from pandas or R : what would have been useful is to see a side by side "this is how you do it in pandas" vs "this is how you do it in atoti". It would decrease the steep learning curve.

This could be extended against other popular libraries as well like plotly. Example of what could be put side by side:
- `store.join` vs `dataframe.merge`
- `cube.query` vs `dataframe.groupby` + `dataframe.unstack` etc
- `cube.visualize` + ui manipulation vs `plotly.graph_objects` + `go.Figure(...)`
- `cube.setup_simulation` vs pandas duplicate data, redefine measures etc

**Comment by fabiencelier** [comment]
It is not code comparison but Dask has a nice global [comparison with Spark](https://docs.dask.org/en/latest/spark.html).

All we have is this point in the [FAQ](https://docs.atoti.io/0.3.1/faq.html#what-is-the-benefit-of-atoti-compared-to-pandas).

We could make it an entire documentation page where we compare use cases, performance and give some code examples.

**Comment by fabiencelier** [comment]
I don't think we can have a mapping from pandas functions to atoti functions such as the [R/pandas mapping](https://pandas.pydata.org/docs/getting_started/comparison/comparison_with_r.html#querying-filtering-sampling) as atoti and pandas are conceptually too different but we could have a mapping of the concepts.

I'll try to make a draft of what we could do, feel free to edit or suggest changes

**Comment by fabiencelier** [comment]
# Comparaison with other tools

## Comparaison with pandas

Since many potential atoti users have some familiarity with pandas, this page is meant to provide some examples of how various pandas operations would be performed using atoti.

If you’re new to atoti, you might want to first read through the Basics tutorial to familiarize yourself with the library.

### Data loading: pandas Dataframes and atoti stores. 

The equivalent of loading data into a pandas Dataframe is loading the data into an atoti store. Most of the pandas loading functions such as pandas.read_csv, pandas.read_parquet or pandas.read_sql have there equivalent in an atoti session: `session.read_csv`, `session.read_parquet`, and `session.read_sql`.

```python
import pandas as pd
df = pd.read_csv("path/to/file.csv")

import atoti as tt
session = tt.create_session()
store = session.read_csv("path/to/file.csv")
```

Both have a `head` method to display their content

```
df.head()

store.head()
```

Additionally a pandas Dataframe can be loaded into a store by using `session.load_pandas`

```python
store = session.read_pandas(df)
```

### Joins

### Data transformations and measures

### Simulations
